feature_tests/java/somelib/src/jmh/java/dev/diplomattest/somelib/OpaqueBench.java

```diff
@@ -0,0 +1,28 @@
+package dev.diplomattest.somelib;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.infra.Blackhole;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+public class OpaqueBench {
```

Comment:
I don't have a shell command to actually run this yet. I only ran it from IntelliJ, but want to keep it as it'll be nice to have to compare to the kotlin backend once